docs/src/security/sso.md

```diff
@@ -131,184 +131,342 @@ lakeFS Cloud uses Auth0 for authentication and thus support the same identity pr
+            client_id: <oidc-client-id>
+            callback_base_url: https://<lakefs.ingress.domain>
+            # the claim name that represents the client identifier in the OIDC provider (e.g Okta)
+            logout_client_id_query_parameter: client_id
+            # the query parameters that will be used to redirect the user to the OIDC provider (e.g Okta) after logout
+            logout_endpoint_query_parameters:
+              - returnTo
+              - https://<lakefs.ingress.domain>/oidc/login
     ```
 
 === "LDAP"
 
-    lakeFS Enterprise provides LDAP authentication by querying the LDAP server for user information and authenticating users based on the provided credentials.
+    lakeFS Enterprise provides direct LDAP authentication by querying the LDAP server for user information and authenticating the user based on the provided credentials.
 
-    **Important:** An administrative bind user must be configured. It should have search permissions for the LDAP server that will be used to query for user information.
+    **Important:** An administrative bind user must be configured. It should have search permissions for the LDAP server.
 
-    To enable working with LDAP, the following configuration must be set:
-
-    1. Replace `providers.ldap.server_endpoint` with your LDAP server endpoint (e.g `ldaps://ldap.company.com:636`)
-    1. Replace `providers.ldap.bind_dn` with the LDAP bind user that has permissions to query your LDAP server
-    1. Replace `providers.ldap.bind_password` with the bind user's password
-    1. Replace `providers.ldap.user_base_dn` with the user base DN to search users in
-    1. Update other LDAP settings as needed for your environment
-
-    **lakeFS Enterprise Configuration:**
+    **Helm Configuration (`values.yaml`):**
 
     ```yaml
-    features: 
-      local_rbac: true
-    auth:
-      ui_config:
-        logout_url: /logout
-        login_cookie_names:
-          - internal_auth_session
-        rbac: internal
-      providers:
+    ingress:
+      enabled: true
+      ingressClassName: <class-name>
+      hosts:
+        - host: <lakefs.ingress.domain>
+          paths:
+            - /
+
+    enterprise:
+      enabled: true
+      auth:
         ldap:
-          server_endpoint: ldaps://ldap.company.com:636
-          bind_dn: uid=bind-user-name,ou=Users,o=org-id,dc=company,dc=com
-          bind_password: your-bind-password
-          username_attribute: uid 
-          user_base_dn: ou=Users,o=org-id,dc=company,dc=com 
-          user_filter: (objectClass=inetOrgPerson)
-          connection_timeout_seconds: 15
-          request_timeout_seconds: 7
+          enabled: true
+          bindPassword: <ldap bind password>
+
+    image:
+      privateRegistry:
+        enabled: true
+        secretToken: <dockerhub-token>
+
+    lakefsConfig: |
+      blockstore:
+        type: local
+      features:
+        local_rbac: true
+      auth:
+        ui_config:
+          login_cookie_names:
```

Comment:
The examples are very verbose. for all examples, remove uneeded things for Helm. For example, `login_cookie_names` irrelevant they don't need to set it in the chart.